file,python,compile,2to3,2to3error
v30.py,python:2.7,no,yes
v33.py,python:2.7,no,yes
v35.py,python:2.7,no,no
v36.py,python:2.7,no,no
v37.py,python:2.7,no,no
v30.py,python:3.3,yes,yes
v33.py,python:3.3,yes,no
v35.py,python:3.3,no,no
v36.py,python:3.3,no,no
v37.py,python:3.3,no,no
v30.py,python:3.4,yes,yes
v33.py,python:3.4,yes,yes
v35.py,python:3.4,no,no
v36.py,python:3.4,no,no
v37.py,python:3.4,no,no
v30.py,python:3.5.2,yes,yes
v33.py,python:3.5.2,yes,yes
v35.py,python:3.5.2,yes,no
v36.py,python:3.5.2,no,no
v37.py,python:3.5.2,no,no
v30.py,python:3.5.3,yes,yes
v33.py,python:3.5.3,yes,yes
v35.py,python:3.5.3,yes,yes
v36.py,python:3.5.3,no,no
v37.py,python:3.5.3,no,no
v30.py,python:3.6.1,yes,yes
v33.py,python:3.6.1,yes,yes
v35.py,python:3.6.1,yes,yes
v36.py,python:3.6.1,yes,no
v37.py,python:3.6.1,no,no
v30.py,python:3.7.0a,yes,yes
v33.py,python:3.7.0a,yes,yes
v35.py,python:3.7.0a,yes,yes
v36.py,python:3.7.0a,yes,yes
v37.py,python:3.7.0a,yes,yes
